core/src/main/java/org/infinispan/xsite/XSiteAdminOperations.java

```diff
@@ -387,13 +391,14 @@ private String takeOffline(String site, Integer afterFailures, Long minTimeToWai
          return incorrectSiteName(site);
       }
 
+      // TODO [pruivo] use mutable attributes to update the take offline settings?
```

Comment:
This PR or future work?

Reply:
Future work, I'm unsure if it works or not since the backup configuration is a list. I want to deprecate the REST endpoint and use the mutable-attribute action. I need to test if I can identify the site's name in the attribute name somehow.

Reply:
Ok. Please track this in a Jira

Reply:
I meant this endpoint: 
https://infinispan.org/docs/stable/titles/rest/rest.html#rest_v2_xsite_tuning_takeoffline_rest

It modifies the cache configuration and it is there because it didn't have mutable attributes. It seems redundant now since this is available:
https://infinispan.org/docs/stable/titles/rest/rest.html#rest_v2_cache_mutable_attribute_set

Reply:
Understood, but with your suggestion the user has to know that our xsite functionality status is stored in the internal CONFIG cache. This is an implementation detail that the user should not care about. The dedicated endpoint provides a higher level abstraction that is more user friendly.

Reply:
Apologies, I had missed the `*-config` suffix when I looked at the endpoint like earlier. The Jira helped, thanks